js/currencycom.js

```diff
@@ -1330,6 +1331,31 @@ module.exports = class currencycom extends Exchange {
         return this.safeString (types, type, type);
     }
 
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'coin': currency['id'],
+        };
+        const response = await this.privateGetV2DepositAddress (this.extend (request, params));
+        //
+        //     { "address":"0x97d64eb014ac779194991e7264f01c74c90327f0" }
+        //
+        return this.parseDepositAddress (response);
+    }
+
+    parseDepositAddress (depositAddress, currency) {
```

Comment:
The `currency` argument must have a default value of `undefined` (see how `parseDepositAddress` is implemented in the other exchanges).